packages/fast-components-styles-msft/src/index.ts

```diff
@@ -122,7 +122,10 @@ export { TypographyStyles };
  */
 export * from "./utilities/acrylic";
 export * from "./utilities/breakpoints";
+export * from "./utilities/colors";

```

Comment:
Do we want to use the export checker utility for this?

Reply:
I don't think it will work because these are all named exports.